tools/push-docker-image.bash

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+version=$1
+
+cd ${version} || exit 1
+
+docker build -t newrelic/c-daemon:latest -t newrelic/c-daemon:${version} .
+
+docker login --username=newrelicphp --password ${DOCKER_HUB_PASSWORD}
+
+docker push newrelic/c-daemon
```

Comment:
I wonder if it's possible to only push dedicated tags? Like `latest` and `${version}`.

It seems this command pushes all tags for the name. That's paranoid, but if that's running on a Jenkins host (or locally too), there might be different tags created before by other jobs (that one doesn't necessarily want to push).